gradle/dependencies.gradle

```diff
@@ -15,18 +15,18 @@
  */
 
 def versions = [
-        kotlin: '1.8.20',
-        ksp: '1.8.20-1.0.11',
-        ktlint: '0.41.0',
-        ktfmt: '0.23',
+        kotlin: '2.1.0',
+        ksp: '2.1.0-1.0.28',
+        ktlint: '1.5.0',
+        ktfmt: '0.54',
         dagger: '2.47',
         butterknife: '10.1.0',
         glide: '4.9.0',
-        gjf: '1.7',
+        gjf: '1.8',
         kotlinpoet: '1.12.0',
         room: '2.1.0',
-        roomCompilerProcessing: '2.6.0-alpha02',
-        dokka: '1.9.10',
+        roomCompilerProcessing: '2.7.0-alpha13',
```

Comment:
Can we bump to a non-alpha?

Reply:
sadly no, this is most recent. it has the necessary set of deps to support the version of kotlin/ksp/jvmMetadata that we want